Makefile

```diff
@@ -72,20 +72,13 @@ cluster-down: delete-config
 prep-config:
 	@echo 'Generating certs and deploying into active cluster...'
 	cat deploy/deployment-base.yaml | sed -e "s|IMAGE|${IMAGE}|g" | tee deploy/deployment.yaml
-	cat deploy/mutatingwebhook.yaml | hack/webhook-patch-ca-bundle.sh > deploy/mutatingwebhook-ca-bundle.yaml
```

Comment:
`deploy/mutatingwebhook-ca-bundle.yaml` is also referenced below in the `delete-config` step, do we need to update there as well?